.nav

```diff
@@ -0,0 +1,4 @@
+getting-started
```

Comment:
Is this file required? It describes the same navigation already available in the [docs.yaml](https://github.com/datastax/nodejs-driver/blob/v3.1.1/docs.yaml#L5-L20).